### ISSUE TITLE
Https support

### ISSUE THREAD

**Issue by b-johnse** [alefragnani/vscode-jenkins-status#22]
I have an enterprise jenkins I would love to view status from, but currently I'm running into a problem around certificates. 
The URL is https, and the feedback the indicator gives currently is: 
```
Job name: Error self signed certificate in certificate chain
URL: https...
Build #.: SELF_SIGNED_CERT_IN_CHAIN
```
So either I'm doing something wrong, or being able to point to a cert in the .jenkins file would be nice

**Comment by alefragnani** [comment]
Hi @b-johnse ,

I'm not sure how this works (I just use basic auth) but I'll take a look how to add this. 

Thanks

**Comment by grzegorzjudas** [comment]
Temporary workaround (but shouldn't be enabled all the time) would be to set environment variable NODE_TLS_REJECT_UNAUTHORIZED to 0.

That'll disable strict TLS checking for all Node applications, so be careful. Better solution @alefragnani , would be to add "strictTls" optionto *.jenkins* file and if it exists, do:

```js
process.env.NODE_TLS_REJECT_UNAUTHORIZED=0;
```
so it's set only for the plugin environment.

**Comment by grzegorzjudas** [comment]
Actually, I have prepared a fix commit. Can I get write access to repo?

**Comment by alefragnani** [comment]
Hi @grzegorzjudas ,

Thanks for your help. You could create a Fork from the repo and send me a Pull Request. 

Do you need any help on this?

**Comment by grzegorzjudas** [comment]
Ah, true - forgot about that possibility. Managed to do that - PR #23

**Comment by alefragnani** [comment]
Closed with #23
